cognite/client/data_classes/extractor_extension/__init__.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import annotations
```

Comment:
Let's name this package extractor_extensions (s at the end) to mirror the space name.

Reply:
Nice catch! Fixed